main.go

```diff
@@ -42,6 +48,7 @@ import (
 var (
 	discoveryInterval = flag.Duration("discovery-interval", 30*time.Second, "How often to refresh discovery cache to pick up newly-installed resources")
 	informerRelist    = flag.Duration("cache-flush-interval", 30*time.Minute, "How often to flush local caches and relist objects from the API server")
+	debugAddr         = flag.String("debug-addr", "localhost:9999", "The address to bind the debug http endpoints")
```

Comment:
Binding to `localhost` by default means it won't be accessible from outside the Pod, doesn't it? Is that intentional? It looks like earlier you had just `":9999"`.

Reply:
Yeah I think you're right I'll change it.